Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -118,6 +119,25 @@
         <ECEnumerator value="2" name="SLOPEDIRECTION" displayLabel="Slope Direction" description="Specialized control vertex that uses slope constraints along a single direction in the spatial interpolation calculation." />
     </ECEnumeration>
 
+    <ECRelationshipClass typeName="CategorySymbolizesClassification" strength="referencing" modifier="Abstract">
+        <Source multiplicity="(0..*)" roleLabel="symbolizes" polymorphic="false">
```

Comment:
This relationship needs to subclass bis:ElementRefersToElements so it can be used at runtime.